governance/chair-proposal-process.md

```diff
@@ -0,0 +1,6 @@
+# Environmental Sustainability WG Chair proposal process
+
+1) Environmental Sustainability WG Co-chairs operate as a team. The Co-Chair team seeks to maintain full coverage of the leadership capabilities across the key qualities of industry experience, hands-on cloud-native and security experience, as well as administrative experience needed to run a Environmental Sustainability WG.
+2) If a clear successor for the exiting Co-Chair exists based on the merit of their contributions ("chop wood, carry water") and relevant experience that will benefit the WG, based on the WG's collective experience over time, the exiting Co-chair may nominate a successor.
```

Comment:
I don't think `chop wood`  or `carry water` is appropriately used in this context.